content/docs/setup/kubernetes/upgrading-istio/index.md

```diff
@@ -255,3 +255,61 @@ spec:
     tls:
       mode: DISABLE
 {{< /text >}}
+
+## Migrating the `RbacConfig` to `ClusterRbacConfig`
+
+The `RbacConfig` has a [bug](https://github.com/istio/istio/issues/8825). The bug reduces the scope of
+the object to be namespace-scoped in some cases. The `ClusterRbacConfig` follows the exact same
+specification as the `RbacConfig` but with correct cluster scope implementation.
+
+1. Replace the custom resource of kind `RbacConfig` with kind `ClusterRbacConfig` if you use
+   [Istio authorization](/docs/concepts/security/#authorization) with custom resource of kind `RbacConfig`:
+
+    A script is provided to automate the migration for you. Depending on if you have the [Istio installation package](/docs/setup/kubernetes/download-release):
```

Comment:
Aside from the passive voice issue: "To automate the migration, we provide a script in the [Istio installation package] ..."
Please replace the non-specific "a script" with the specific filename: "the `convert_RbacConfig_to_ClusterRbacConfig.sh` script. Do not structure the content as an not-ordered list.
The text in lines 268-288 should be a series of steps.
Since you do not know whether the reader has downloaded the installation package, the flow should be:
1. Download the installation package or Download the script 
   <command to download the package or the script>
2. Run the script
   <command to run the script>
3. Verify the migration is complete
   <command to verify the migration was completed.